README.md

```diff
@@ -1,3 +1,10 @@
+[![Unit tests](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/workflows/Unit%20tests%20workflow/badge.svg)](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/actions)
+[![Integration tests](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/workflows/E2E%20tests%20workflow/badge.svg)](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/actions)
+[![codecov](https://codecov.io/gh/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/branch/master/graph/badge.svg)](https://codecov.io/gh/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin)
+[![Documentation](https://img.shields.io/badge/api-reference-blue.svg)](https://opendistro.github.io/for-elasticsearch-docs/docs/ad/api/)
```

Comment:
Not sure if we should add API link for frontend. How about we add ODFE AD doc here https://opendistro.github.io/for-elasticsearch-docs/docs/ad/

Reply:
sure.